bark_ml/observers/alt_nearest_observer.hpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) 2019 fortiss GmbH, Patrick Hart, Julian Bernhard, Klemens Esterle, Tobias Kessler
```

Comment:
do not rename it to `alt` this is not international

Reply:
What is different in your implementation?

Reply:
merged the observers and updated the test